mava/wrappers/jumanji.py

```diff
@@ -217,3 +219,101 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             )
 
         return spec
+
+
+class CleanerWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Cleaner environment.
+
+    Do not use the AgentID wrapper with this env, it has implicit agent IDs.
+    """
+
+    def __init__(self, env: Cleaner, has_global_state: bool = False):
+        super().__init__(env)
+        self.has_global_state: bool = False
+
+    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
+        """Modify the timestep for the Cleaner environment."""
+
+        def create_agents_view(grid: chex.Array, agents_locations: chex.Array) -> chex.Array:
```

Comment:
Can we add some shape comments throughout this method above the lines where you tile/repeat, e.g `(A, Width, Height)`

Also can you please add docs to explain how we create the agents view

Reply:
+1 on this please :pray: